src/utils/scroll-lock.js

```diff
@@ -131,7 +131,7 @@ export default {
     if (!isIosDevice()) {
       simpleLock();
     } else {
-      advancedLock(targetElement);
```

Comment:
I may be misunderstanding what's happening here, but if `targetElement` may be null/undefined, shouldn't we just change the `else` to `else if (targetElement)` and/or fix the places that might be calling `lockScroll` without an element?

It seems a little odd to me that we would even try calling the method without an element.

Reply:
Right, but if we locked previously and try to unlock, even if the lock target is no longer in the DOM, we wont be able to. But we still want to clean up the rest (there are some even listeners attached to the body).

What I can do is just inside the `advancedLock` check if `!targetElement` and apply code conditionally.